tests/unit/amo/components/TestReportAbuseButton.js

```diff
@@ -304,4 +309,16 @@ describe(__filename, () => {
     // method was called.
     sinon.assert.called(fakeEvent.preventDefault);
   });
+
+  it('renders an error if one exists', () => {
+    const errorHandler = createStubErrorHandler();
+    const { store } = dispatchClientMetadata();
+    store.dispatch(setError({
+      error: new Error('something bad'),
+      id: errorHandler.id,
+    }));
+    const root = renderMount({ errorHandler, store });
```

Comment:
I don't see any advantage to doing a mounted render and it has several disadvantages for future maintenance. I would like to see a shallow render where you just check that the component exists:

```js
expect(root.find(ErrorList)).toHaveLength(1);
```

Reply:
Hmm, this was my only change request but github doesn't show it in the context of my review for some reason.